parsec/core/cli/run.py

```diff
@@ -55,3 +63,173 @@ def run_mountpoint(config, device, mountpoint, timestamp, **kwargs):
         config = config.evolve(mountpoint_base_dir=Path(mountpoint))
     with cli_exception_handler(config.debug):
         trio_run(_run_mountpoint, config, device, timestamp)
+
+
+async def _import_file(workspace_fs: WorkspaceFS, src: FsPath, dest: FsPath):
```

Comment:
All rsync related functions should probably go to their own module (e.g. `parsec/core/cli/rsync.py`)